src/Aspire.Dashboard/Model/KnownPropertyLookup.cs

```diff
@@ -27,7 +27,6 @@ public KnownPropertyLookup(IStringLocalizer<Resources.Resources> loc)
             new(KnownProperties.Resource.StartTime, loc[nameof(ResourcesDetailsStartTimeProperty)]),
             new(KnownProperties.Resource.StopTime, loc[nameof(ResourcesDetailsStopTimeProperty)]),
             new(KnownProperties.Resource.ExitCode, loc[nameof(ResourcesDetailsExitCodeProperty)]),
-            new(KnownProperties.Resource.HealthState, loc[nameof(ResourcesDetailsHealthStateProperty)])
```

Comment:
We still want to display calculated health status in resources propertries in details view